### PR TITLE
New UI

### PR DESCRIPTION
**Pull request by hamza94max**

### CODE REVIEW

File: app/src/main/java/com/islamey/haamzaa/WAZAKER/data/DataSets/FortyDataSet.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
-package com.hamza.Wazaker.Data.DataSets;
+package com.islamey.haamzaa.WAZAKER.data.DataSets;
 
-import com.hamza.Wazaker.Data.Models.Hadith;
+import com.islamey.haamzaa.WAZAKER.data.Models.Hadith;
```
Comment (MahmoudMabrok) [inline app/src/main/java/com/islamey/haamzaa/WAZAKER/data/DataSets/FortyDataSet.java:3]: please update package name to be small/lower case not Upper case

**Review by MahmoudMabrok — CHANGES_REQUESTED** [review]
please check requested changes

File: app/build.gradle [hunk 2]
```diff
@@ -9,12 +9,12 @@ android {
...
     defaultConfig {
-        applicationId "com.hamza.wazkar"
+        applicationId "com.islamey.wazkar"
         minSdkVersion 16
         //noinspection OldTargetApi,ExpiredTargetSdkVersion
         targetSdkVersion 31
-        versionCode 15
-        versionName "1.0.1"
+        versionCode 1
+        versionName "1.0.0"
```
[the remark this answers is not in the document]
Comment (hamza94max) [reply app/build.gradle:17]: No, It's an old change

File: app/src/main/java/com/islamey/haamzaa/WAZAKER/data/DataSets/AzkarListDataset.kt [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        azkarList.add(AzkarListModel("أذكار النوم"))
+        azkarList.add(AzkarListModel("أذكار الإستيقاظ من النوم"))
+        azkarList.add(AzkarListModel("أذكار عظيمة"))
+        azkarList.add(AzkarListModel("دعاء السفر"))
+        azkarList.add(AzkarListModel("دعاء ختم القرآن الكريم"))
+        azkarList.add(AzkarListModel("فضل الدعاء"))
+        azkarList.add(AzkarListModel("فضل تلاوة القرآن الكريم"))
+        azkarList.add(AzkarListModel("فضل الأذكار"))
+        azkarList.add(AzkarListModel("الرقية الشرعية"))
+        azkarList.add(AzkarListModel("أدعية للمتوفّي"))
```
[the remark this answers is not in the document]
Comment (hamza94max) [reply app/src/main/java/com/islamey/haamzaa/WAZAKER/data/DataSets/AzkarListDataset.kt:27]: You mean JSON or what ?

**Pull request closed without merging**